.github/workflows/update-flatpak-docs.yml

```diff
@@ -0,0 +1,58 @@
+name: Update Flatpak and Flatpak Builder docs
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'
+  workflow_dispatch: {}
+
+jobs:
+  archive:
+    name: Update Flatpak and Flatpak Builder docs
+    runs-on: ubuntu-latest
```

Comment:
The biggest problem here is Ubuntu. They don't update, so this will happen once every 6 months I guess.

Reply:
Ah I missed the PPA :)